ee/src/main/java/org/jboss/as/ee/subsystem/ManagedExecutorServiceResourceDefinition.java

```diff
@@ -70,16 +78,18 @@
 
     public static final SimpleAttributeDefinition CONTEXT_SERVICE_AD =
             new SimpleAttributeDefinitionBuilder(CONTEXT_SERVICE, ModelType.STRING, true)
-                    .setAllowExpression(false)
+                    .setAllowExpression(true)
```

Comment:
This change requires a new model version and transformers to reject models/operations containing expressions for older versions.